ui/src/app/views/resource-card/component.spec.ts

```diff
@@ -88,30 +89,6 @@ describe('ResourceCardComponent', () => {
     updateAndTick(fixture);
     expect(fixture.debugElement.query(By.css('.notebook-card'))).toBeTruthy();
   }));
-
-  // Actual actions are tested in cohort list/notebook list specs...
-  // CURRENTLY those tests don't technically test these actions, but will once
-  // planned refactoring takes place -US
-  it('should have correct options in menu when it is a cohort', fakeAsync(() => {
-    const de = fixture.debugElement;
-    simulateClick(fixture, de.query(By.css('.dropdown-toggle')));
-    const actionItems = de.queryAll(By.css('.action-item'))
-      .map((item) => item.nativeElement.innerText.trim());
-    ['Clone', 'Edit', 'Review', 'Delete'].forEach((action) => {
-      expect(actionItems).toContain(action);
-    });
-  }));
-
-  it('should have correct options in menu when it is a notebook', fakeAsync(() => {
-    setNotebookResource(fixture, component);
-    const de = fixture.debugElement;
-    simulateClick(fixture, de.query(By.css('.dropdown-toggle')));
-    const actionItems = de.queryAll(By.css('.action-item'))
-      .map((item) => item.nativeElement.innerText.trim());
-    ['Rename', 'Clone', 'Delete'].forEach((action) => {
-      expect(actionItems).toContain(action);
-    });
-  }));
```

Comment:
NB: Deleted this because this logic is now straightforward to verify by inspecting the code, and the other tests already do more thorough testing of the underlying behavior.